lib/runtime/genesis.go

```diff
@@ -26,7 +26,7 @@ func NewTrieFromGenesis(gen genesis.Genesis) (tr trie.Trie, err error) {
 			ErrGenesisTopNotFound, gen.Name)
 	}
 
-	tr, err = trie.LoadFromMap(keyValues)
+	tr, err = trie.LoadFromMap(keyValues, trie.V0)
```

Comment:
Why V0 here?

Reply:
I believe this is because we are loading it from genesis and assuming that genesis is using V0 but if I'm not wrong paseo uses V1 on genesis right?

Maybe we should get the version from the genesis configuration

Reply:
Yeah, the `LoadFromMap` is not 100% accurate to V1 since we need the infos of the encoded trie and not only the keys and values to decode a trie from maps, that's why it is by default `trie.V0`. But for some **test cases** we can use the arg `trie.V1`.

@dimartiro paseo uses V0

Reply:
Ok we can say that is safe to use V0 until we start using a genesis with V1 set, maybe we can add this as a "known issue"? 🤔

Reply:
@dimartiro yes!